api/objects.proto

```diff
@@ -342,6 +342,14 @@ message Cluster {
 	// reject joining the cluster.  Nodes that report themselves to be non-FIPS
 	// should be rejected from the cluster.
 	bool fips = 10 [(gogoproto.customname) = "FIPS"];
+
+	// This flag specifies default subnet pools for global scope networks. If subnet
+	// size is not specified then default value 24 will be used. If unspecified,
+	// Docker will use the predefined subnets as it works on older releases.
+	repeated string defaultAddressPool = 11;
+
+	// This flag specifies default subnet size for global scope networks
+	uint32 subnetSize = 12;
 }
```

Comment:
If we aren't going to change the name of the field  (again recommending something like `subnetMaskLen`, then at the very least I think we should add documentation here saying something like:  "This flag specifies the default subnet size of global scope networks by giving the length of the subnet masks for every such network".